linux_os/guide/services/ssh/service_sshd_disabled/kubernetes/shared.yml

```diff
@@ -0,0 +1,16 @@
+---
+# platform = multi_platform_fedora,multi_platform_rhel,multi_platform_rhcos
+# reboot = true
+# strategy = restrict
+# complexity = low
+# disruption = low
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    systemd:
+      units:
+      - name: sshd.service
+        enabled: false
```

Comment:
Why not use the `shared/templates/service_disabled/kubernetes.template` template? It looks like the template also masks the service and disables the socket to prevent socket activation later on.

Reply:
that's a good idea, let me change that